test/specs/conditions/empty.yaml

```diff
@@ -0,0 +1,3 @@
+condition:
+
+vcl: ''
```

Comment:
That's not valid. `if()` in VCL would give you an error.

Reply:
Fair enough, I removed that file